lib/active_model/serializer/caching.rb

```diff
@@ -208,46 +199,44 @@ def object_cache_key(serializer, adapter_instance)
         end
       end
 
-      def cached_attributes(fields, cached_attributes, adapter_instance)
+      ### INSTANCE METHODS
+      def fetch_attributes(fields, cached_attributes, adapter_instance)
```

Comment:
renamed cached_attributes which calls cache_check to fetch_attributes which calls fetch. 

Naming goes all the way back https://github.com/rails-api/active_model_serializers/pull/810/files https://github.com/rails-api/active_model_serializers/pull/693/files

I have more simplifications in mind, in particular moving the cache code outside of a serializer, or perhaps only included when `cache` is called.